dockerfiles/cli/cmd_start.sh

```diff
@@ -53,8 +53,13 @@ cmd_start() {
   # Start Codenvy
   # Note bug in docker requires relative path, not absolute path to compose file
   info "start" "Starting containers..."
-  log "docker_compose --file=\"${REFERENCE_CONTAINER_COMPOSE_FILE}\" -p=$CHE_MINI_PRODUCT_NAME up -d >> \"${LOGS}\" 2>&1"
-  docker_compose --file="${REFERENCE_CONTAINER_COMPOSE_FILE}" \
-                 -p=$CHE_MINI_PRODUCT_NAME up -d >> "${LOGS}" 2>&1
+  COMPOSE_UP_COMMAND="docker_compose --file=\"${REFERENCE_CONTAINER_COMPOSE_FILE}\" -p=\"${CHE_MINI_PRODUCT_NAME}\" up -d"
+
+  if [ "${CODENVY_DEVELOPMENT_MODE}" != "on" ]; then
+    COMPOSE_UP_COMMAND+=" >> \"${LOGS}\" 2>&1"
+  fi
+
+  log ${COMPOSE_UP_COMMAND}
+  eval ${COMPOSE_UP_COMMAND}
```

Comment:
maybe we can improve it like this, my only motivation here is that due to frequent changes in CLI guys fail to start codenvy and they come to me and saying it didn't work. if they will see error message on console it will be much easier for them to figure out where is a problem.